notebook.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<p>Since you already provide a conda environment, I would suggest to add this dependency there instead of installing it on the fly. At least on the system that I am using this can create problems (HPC where I do not have writing rights for the directory pip wants to access)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/reproduce-deep-prior-4Dvar/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Agreed. It's a good practise to declare all dependencies in the conda env instead! </p>

Reply:
closing conversation, resolved